tests/cases/issues/issue_311/test_issue_311/test_issue_311.py

```diff
@@ -0,0 +1,11 @@
+from pytest_cases import fixture, parametrize_with_cases
+
+
+@fixture
+@parametrize_with_cases('arg', cases='cases')
+def function_scoped(arg):
```

Comment:
Same comment as for the class-scoped: could you imagine a way to include this fixture in the test too, and in particular a way to be sure that the scope is correctly taken into account for all 3 ?